AlphaWallet/Sell/ViewControllers/EnterSellTokensCardPriceQuantityViewController.swift

```diff
@@ -94,9 +94,13 @@ class EnterSellTokensCardPriceQuantityViewController: UIViewController, TokenVer
         dollarCostLabel.translatesAutoresizingMaskIntoConstraints = false
 
         pricePerTokenField.translatesAutoresizingMaskIntoConstraints = false
-        ethPrice.subscribe { [weak self] value in
-            if let value = value {
-                self?.pricePerTokenField.ethToDollarRate = value
+        if config.chainID == 100 {
```

Comment:
Better to use a  `switch` on `config.server` instead of if-else and `config.chainID`. Bit more code, but it's strong type checking (harder to key in or change the `100` to an invalid value) and it's easier to spot special handling of `RPCServer.xDai` by looking up references rather than search for `100`.